bin/hisensetv/__main__.py

```diff
@@ -25,7 +27,7 @@ def main():
         "--get",
         action="append",
         default=[],
-        choices=["sources", "volume", "state"],
```

Comment:
Why has the state choice been removed?

Reply:
must have been an issue when I copied from the original. I just did a compare in notepad++ to make sure i didnt miss any others, and have put state back in.